packages/data-connect/karma.conf.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
```

Comment:
Probably should check this year on all files.

Reply:
Checked! Thanks!

Reply:
Sorry, reviving this comment. Many of these files have copyrights from varying years. Shouldn't they all be 2024?